app/build.gradle.kts

```diff
@@ -126,7 +126,8 @@ dependencies {
     // Network client
     implementation("com.squareup.okhttp3:okhttp:${Versions.OKHTTP}")
     implementation("com.squareup.okhttp3:logging-interceptor:${Versions.OKHTTP}")
-    implementation("com.squareup.okio:okio:2.6.0")
+    implementation("com.squareup.okhttp3:okhttp-dnsoverhttps:${Versions.OKHTTP}")
+    implementation("com.squareup.okio:okio:2.7.0")
```

Comment:
why did you update okhttp?

Reply:
accidetally got included XD

Reply:
and i change it to 2.7.0, i think it was 2.7.0 too on ino/tachi :tehe: